clients/privacy-center/config/config.json

```diff
@@ -32,19 +32,20 @@
   "consent": {
     "consentOptions": [
       {
-        "fidesDataUseKey": "third_party_sharing",
-        "name": "Do not sell my personal information",
-        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit. Lorem ipsum dolor sit amet, consectetur.",
-        "highlight": true,
-        "url": "https://example.com/privacy#data-sales"
+        "fidesDataUseKey": "advertising",
+        "name": "Advertising / Data Sharing",
+        "description": "We may use some of your personal information for advertising performance analysis and audience modeling for ongoing advertising which may be interpreted as 'Data Sharing' under some regulations.",
+        "url": "https://example.com/privacy#advertising",
+        "default": true,
+        "highlight": false
```

Comment:
I wanted to edit this to also be more realistic examples. Unrelated change, but will be welcome in general testing.